code/modules/cargo/exports/sheets.dm

```diff
@@ -10,46 +10,46 @@
 // Hides
 
 /datum/export/stack/skin/monkey
-	cost = 50
+	cost = 200
 	unit_name = "monkey hide"
 	export_types = list(/obj/item/stack/sheet/animalhide/monkey)
 
 /datum/export/stack/skin/human
-	cost = 100
+	cost = 1500
```

Comment:
requires human sacrifices

Reply:
Note that human skin was at 2000 before the bounty PR

Reply:
didnt revert to 100% to include bonus money made with bounties

Reply:
I wasn't saying you should, I just wanted to point it out for others.